packages/lib/util.ts

```diff
@@ -1,6 +1,18 @@
 import config from 'config'
 import crypto from 'crypto'
+import fs from 'fs'
+
 const algorithm = 'aes-256-ctr'
+const iv = ((buildFilePath = '../config/build.json', ivLength = 16): string | Buffer => {
```

Comment:
> if you encrypt something with one build & later try to decrypt the result with new build - you wouldn't be able to do that, because the IV value is changes from build to build

I didn't thought about that – great research. I agree, your implementation isn't just more save it's also more accurate and needs less code and logic. Awesome.

Reply:
I've updated my code as well – so VSF can decide which to merge.